jenkins/bootstrap.py

```diff
@@ -800,14 +800,15 @@ def job_script(job, scenario, extra_job_args):
         config = json.loads(fp.read())
     if job.startswith('pull-security-kubernetes-'):
         job = job.replace('pull-security-kubernetes-', 'pull-kubernetes-', 1)
-    if job not in config:
-        if not scenario:
-            raise ValueError('cannot find scenario for job', job)
-        cmd = test_infra('scenarios/%s.py' % scenario)
-    else:
+    config_json_args = []
+    if job in config:
         job_config = config[job]
-        cmd = test_infra('scenarios/%s.py' % job_config['scenario'])
-    return [cmd] + job_args(job_config.get('args', []) + extra_job_args)
+        scenario = job_config['scenario']
+        config_json_args = job_config.get('args', [])
+    elif not scenario:
+        raise ValueError('cannot find scenario for job', job)
+    cmd = test_infra('scenarios/%s.py' % scenario)
+    return [cmd] + job_args(config_json_args + extra_job_args)
```

Comment:
on a second thought - `--scenario` probably should ignore `config_json_args`

Reply:
this seems orthogonal? we should probably migrate bit by bit out of config.json

Reply:
we also need to start labeling sig-owners on the jobs themselves, I'll do this later.

Reply:
because I don't have a `config.json` internally, luckily the bazel-test job does not have any config.json args, but it might not work for stuff like e2e-gce

Reply:
this will gracefully use [] for `config_json_args` in that case